lib/cc/pull_requests.rb

```diff
@@ -1,8 +1,15 @@
 class CC::PullRequests < CC::Service
+  VALID_TOKEN_MESSAGE = "Access token is valid.".freeze
+  CANT_UPDATE_STATUS_MESSAGE = "Access token is invalid - can't update status.".freeze
+
   def receive_test
     setup_http
 
-    receive_test_status
+    if able_to_update_status?
+      { ok: true, message: VALID_TOKEN_MESSAGE }
+    else
+      { ok: false, message: CANT_UPDATE_STATUS_MESSAGE }
+    end
```

Comment:
This seems less useful than the default behavior which includes the external error message: https://github.com/codeclimate/codeclimate-services/blob/master/lib/cc/service/invocation/with_error_handling.rb

What do you think about reducing this diff a bit further or is there a functional reason for these changes?